projects/1.20/assets/productivebees/productivebees/patchouli_books/guide/zh_cn/entries/blocks/gene_indexer.json

```
@@ -0,0 +1,16 @@
+{
+    "name": "基因检索器",
+    "icon": "productivebees:gene_indexer",
+    "category": "productivebees:blocks",
+    "pages": [
+        {
+            "type": "patchouli:spotlight",
+            "item": "productivebees:gene_indexer",
+            "text": "你是否淹没在基因样本中？你还在通过通过合并基因来获得 RSI吗？那么这个方块就是为你准备的！"
```

```suggestion
            "text": "基因样本是不是多到完全无法招架？长时间手动组合基因有没有让你劳累不堪？那么这个方块就是为你准备的！"
```
RSI可能指重复性劳损（Repetitive strain injury）